docs/wiki/protocol/architecture.md

```diff
@@ -0,0 +1,100 @@
+<h2>What is the Ethereum ?</h2>
+<p>Ethereum is an open source, globally, public and decentralized distributed ledger system.</p>

```

Comment:
This document should cover protocol architecture, discuss its logic, rationale, contains etc. There is no reason to cover basic questions like What is Ethereum or try to define a blockchain.

Reply:
I think without understanding these two definition, anybody cant understand protocol structure.